src/pages/registration/resource_type_tab/sub_type_forms/media_types/MediaContributionPeriodicalForm.tsx

```diff
@@ -0,0 +1,108 @@
+import { ErrorMessage, Field, FieldProps } from 'formik';
+import { useTranslation } from 'react-i18next';
+import { Box, TextField, Typography } from '@mui/material';
+import { ResourceFieldNames } from '../../../../../types/publicationFieldNames';
+import { dataTestId } from '../../../../../utils/dataTestIds';
+import { DoiField } from '../../components/DoiField';
+import { JournalField } from '../../components/JournalField';
+import { PublicationChannelType } from '../../../../../types/registration.types';
+
+export const MediaContributionPeriodicalForm = () => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <DoiField />
+
+      <JournalField
+        confirmedContextType={PublicationChannelType.MediaContributionPeriodical}
+        unconfirmedContextType={PublicationChannelType.UnconfirmedMediaContributionPeriodical}
+      />
+
+      <Box
```

Comment:
kanskje flytt alt dette til en ny komponent i stedet for å duplisere da det er likt i Journal?

Reply:
døøøøn